cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/LocalDataLayer.java

```diff
@@ -174,7 +174,7 @@ public static LocalDataLayer from(Map<String, String> options)
                       .filter(StringUtils::isNotEmpty)
                       .collect(Collectors.toSet()),
                 SchemaFeatureSet.initializeFromOptions(options),
-                getBoolean(options, lowerCaseKey("useSSTableInputStream"), false),
+                getBoolean(options, lowerCaseKey("useBufferingInputStream"), getBoolean(options, lowerCaseKey("useSSTableInputStream"), false)),
```

Comment:
this is a behavior change. We should support the old property also for backwards compatibility

Reply:
The code here supports both the old and new prop.

Reply:
ah yes, I completely missed that. Thanks!